design/src/high-level/creative-automation-case-studies.md

```diff
@@ -0,0 +1,282 @@
+
+Fun, silly, intuitive components.
+
+## [Opus Magnum](https://store.steampowered.com/app/558990/Opus_Magnum/)
+
+**Conclusion:** Awesome creative automation, but these are mostly just contraptions without a larger purpose.
+
+### Interesting Goals
+
+Pre-baked puzzles, but with really explicit and interesting multi-objective optimization:
+
+- space
+- cycles
+- cost
+- instructions
+
+These are made explicit to the player, and histogram leaderboards encourage competitive optimization.
+
+### Distinct Paths
+
+Fantastic: lots of recipe paths, fantastic constraints and tradeoffs on how to move things around.
+
+### Low Pressure
+
+Great, it's a puzzle game.
+
+### Compelling Aesthetics
+
+Clean and clear aesthetics, cohesive if mostly abstract theme, great whirring and moving contraptions.
+
+## [Oxygen Not Included](https://store.steampowered.com/app/457140/Oxygen_Not_Included/)
+
+**Conclusion:** Good but not great creative automation. Automation options take a long time to unlock. Too many challenges have Best Solutions.
+
+### Interesting Goals
+
+Challenges are largely "satisfy", not "optimize".
+Figuring out how to:
+
+- get food
+- get water
+- get power
+- produce oxygen
+- manage excess heat
+
+is interesting as a new player, but can be dull once the mechanics are understood.
+Very little incentive to actually fine tune working processes.
+
+### Distinct Paths
+
+Lots of good options for each path, mostly restricted by world gen.
+
+Some setups are simpler / more efficient / more reliable (hi Self-Power-Oxygen-Machines) and end up completely dominating over more interesting ones (like Morb farming).
+
+Duplicant behavior can be hard to understand and optimize for in frustrating ways.
+
+Limited tools for genuine automation until quite late in the game.
+
+### Low Pressure
+
+Great: intermittent emergency failures don't detract from long periods of tweaking and daydreaming.
+
+Generous pause functionality (you can pan and investigate) is essential to making this work.
+
+### Compelling Aesthetics
+
+## [Minecraft](https://www.minecraft.net/en-us)
+
+**Conclusion:** Solid creative automation. Often hard to find goals.
+
+### Interesting Goals
+
+### Distinct Paths
+
+### Low Pressure
+
+### Compelling Aesthetics
+
+## [Terraria](https://store.steampowered.com/app/105600/Terraria/)
+
+**Conclusion:** Limited tools for creative automation. Very little point in doing so.
+
+### Interesting Goals
+
+### Distinct Paths
+
+### Low Pressure
+
+### Compelling Aesthetics
+
+### Atrio: The Dark Wild
+
+**Conclusion:** Great aesthetics. Tools to add diversity to automation take a very long time to unlock and goals are hard to come by.
+
+#### Interesting Goals
+
+### Distinct Paths
+
+### Low Pressure
+
+### Compelling Aesthetics
+
+## [Shapez.io](https://store.steampowered.com/bundle/21489/shapez_Full_Edition/)
+
+**Conclusion:** Limited options for performing each step. Really uninspiring aesthetics.
```

Comment:
Interesting you feel this way! I personally loved Shapez.io's art style: it really scratched my "ooh, i want to smush abstract things into other abstract things" itch, which basically comes out of: "I like thinking about how factory games generalize/abstract"

I also think some of the highly zoomed out screenshots of shapez.io are pretty

My taste is admittedly poorly developed.

Reply:
Ugly and uninspiring aren't the same thing :D I think it's quite pretty, but doesn't really push me to like... make a factory focused on circles?

Reply:
Whereas when playing Factorio it's really interesting and compelling to try for an "all steam power" build.

Reply:
Ahh, that's what you mean by "uninspired!"